### ISSUE TITLE
Disable dependabot

### ISSUE THREAD

**Issue by UebelAndre** [bazelbuild/rules_rust#2135]
Seems dependabot is back? Can it be disabled again (https://github.com/bazelbuild/rules_rust/issues/1390)?

**Comment by UebelAndre** [comment]
cc @krasimirgg @scentini

**Comment by UebelAndre** [comment]
ping @scentini I think there's some checkbox in the repo settings you can use to disable this.

**Comment by scentini** [comment]
Sorry for the delay @UebelAndre . It seems that we have disabled dependabot apart from security vulnerability updates, and I'd rather not disable those. Is getting rid of the recent PRs a matter of repinning the dependencies?

**Comment by UebelAndre** [comment]
> Sorry for the delay @UebelAndre . It seems that we have disabled dependabot apart from security vulnerability updates, and I'd rather not disable those. Is getting rid of the recent PRs a matter of repinning the dependencies?

Yeah, re-pinning would probably address these. But the issue with Dependabot is that the PR it's suggesting is not sufficient for bumping dependencies. In either the case of `crates_repository` or `crates_vendor` Bazel will need to be re-run. It would be awesome if Dependabot could be configured to do the right thing. But it's otherwise just noise to me.
